bindata/cert-manager-deployment/controller/cert-manager-deployment.yaml

```diff
@@ -27,31 +27,34 @@ spec:
         app.kubernetes.io/component: controller
         app.kubernetes.io/instance: cert-manager
         app.kubernetes.io/name: cert-manager
-        app.kubernetes.io/version: v1.9.1
+        app.kubernetes.io/version: v1.10.2
     spec:
       containers:
         - args:
             - --v=2
             - --cluster-resource-namespace=$(POD_NAMESPACE)
             - --leader-election-namespace=kube-system
-          command:
```

Comment:
As per discussion today, lets add this logic back.

Reply:
Added in https://github.com/openshift/cert-manager-operator/pull/98/commits/f381426fe510f7db95adec37253bf6d2dad6d970.